pkg/cloudprovider/plugins/aws/aws_ec2.go

```diff
@@ -127,7 +127,7 @@ func (ec2Cfg *ec2ServiceConfig) getManagedVpcIDs() map[string]struct{} {
 		awsPluginLogger().V(4).Info("Cache snapshot nil", "type", providerType, "account", ec2Cfg.accountNamespacedName)
 		return vpcIDsCopy
 	}
-	vpcIDsSet := snapshot.(*ec2ResourcesCacheSnapshot).managedVpcIDs
+	vpcIDsSet := snapshot.(*ec2ResourcesCacheSnapshot).managedVpcIds
```

Comment:
nit: vpcIdsSet, vpcIdsCopy. 
If you are renaming the variables, please rename all the above.

Reply:
Fixed all in azure_compute.go and aws_ec2.go .